docs/pages/components/button.md

```diff
@@ -8,116 +8,67 @@ permalink: components/button.html
+Buttons allow users to perform actions. All the buttons require base class `fd-button` and additional modifier for all the flavours.
 
 {% capture button %}
-<button class="fd-button fd-button--emphasized">Emphasized Button</button>
-<button class="fd-button">Regular Button</button>
-<button class="fd-button fd-button--light">Light Button</button>
+<button class="fd-button">Create</button>
+<button class="fd-button sap-icon--cart"></button>
+<button class="fd-button fd-button--emphasized ">Save</button>
 {% endcapture %}
 {% include display-component.html component=button %}
 
-## Button Types
+{: .docs-intro}
+There are 4 button types:
+
+- **Simple button** for one action.
+- **Toggle button** for switching between different states.
+- **Segmented button** for a group of options.
+- **Menu button** with a group of actions.
 
-There are five button types that can be combined with the emphasis styles.
-- **Action Button**: The default button
-- **Standard Button**: Neutral or informative color
-- **Positive Button**: Used for positive actions such as approved, ok, yes.
-- **Medium Button**: Used for warnings or inside message strips
-- **Negative Button**:Used for negative actions such as decline, cancel, no.
+<br>
 
+## Design Types
+
+- **Default Button**
+- **Emphasized Button** Used for primary action
+- **Ghost Button** It is used for secondary actions or primary button in case there is already a page primary button
+- **Standard Button** Neutral or informative (secondary) actions
+- **Positive Button** Used for positive (secondary) actions
+- **Negative Button** Used for negative (secondary) actions
+- **Attention Button**
+- **Transparent Button** Used for secondary or negative path actions
 
 {% capture button %}
-<button class="fd-button">Action Button</button>
+<button class="fd-button">Default Button</button>
+<button class="fd-button fd-button--emphasized">Emphasized Button</button>
+<button class="fd-button fd-button--ghost">Ghost Button</button>
 <button class="fd-button fd-button--standard">Standard Button</button>
 <button class="fd-button fd-button--positive">Positive Button</button>
 <button class="fd-button fd-button--negative">Negative Button</button>
+<button class="fd-button fd-button--attention">Attention Button</button>
+<button class="fd-button fd-button--transparent">Transparent Button</button>
 {% endcapture %}
 {% include display-component.html component=button %}
 
+<br>
 
-## Button Sizes
-There are two sizes.
+## Button Types
 
-> The compact size is only used on desktop and it is full size when used on a touch device.
-
-{% capture button %}
-<button class="fd-button">Default</button>
-<button class="fd-button fd-button--compact">Compact</button>
-{% endcapture %}
-{% include display-component.html component=button %}
+### Simple Button
 
+It is recommended to use simple button for specific actions, such as _Create, Edit, Save, Approve, Reject, Accept, Decline, OK, Cancel_
```

Comment:
extra space required at the end of Cancel_ (it is stuck to "All in the line below"